InWorldz/InWorldz.RemoteAdmin/RemoteAdminPlugin.cs

```diff
@@ -191,7 +191,7 @@ public object RegionShutdownHandler(IList args, IPEndPoint remoteClient)
             {
                 m_log.ErrorFormat("[RADMIN] Shutdown: failed: {0}", e.Message);
                 m_log.DebugFormat("[RADMIN] Shutdown: failed: {0}", e.ToString());
-                throw e;
+                throw;
```

Comment:
Semantic change.  Not important:

This is only called by `RemoteAdmin::XmlRpcCommand` which uses a generic `try{...}catch(Exception e){...}` and will handle it fine.